.gitignore

```diff
@@ -48,6 +48,7 @@ node_modules
 *.log
 .nvm
 /danger/node_modules/
+scripts/.packager.env
```

Comment:
Why do we need this?

Reply:
When I was running the build locally, that file was generated and I didn't want to commit any build artifacts. It's unrelated to my change and I can remove it if necessary.